include/yaclib/log_config.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <functional>
+#include <string_view>
+
+using callback_type = std::function<void(std::string_view, std::string_view, size_t, std::string_view)>;
```

Comment:
```suggestion
using LogCallbackType = void(*)(std::string_view, std::string_view, size_t, std::string_view);
```

Reply:
I guess with that nothing can be captured inside? Like file stream for file logging?

Reply:
Its ok, in general they can use static variables, request changes this approach on our side